docs/host/host.md

```diff
@@ -33,30 +33,102 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
 
 ## Deleting a Node
 
-Deleting a node is done in two phases:
+:::caution
 
-1. Delete the node from Harvester
-    - Go to the **Hosts** page
-    - On the node you want to modify, click **⋮ > Delete**
+To delete a node from a Harvester cluster, it's very important to consider where the remaining nodes in the cluster have enough computing resources and storage capacity to accept the workload from the deleting node. If the remaining nodes don't have enough resources, removing a node might cause some VMs to fail to start or some volumes to degrade. You can check the **Hosts** page for the current resource usage in the cluster, and check if the remaining nodes in the cluster can maintain enough replica counts for all volumes.
 
-2. Uninstall RKE2 from the node
-    - Login to the node as root
-    - Run `rke2-uninstall.sh` to delete the whole RKE2 service.
+:::
 
-:::caution
+### 1. Check if the node can be removed from the cluster
+
+If you plan to remove a control plane node, you must consider the availability of the remaining control plane nodes. We'll discuss this topic in two cases:
+
+- There are 3 control plane nodes in the cluster
+
+    You can remove a control plane node in this case, but you need to ensure there are worker nodes in the cluster. A worker node will be promoted to a control plane node after you delete the control plane node. Note, that the worker node to promote is selected randomly before Harvester v1.3.0. Starting from Harvester v1.3.0, Harvester supports specifying a role for a new-added node. If you prefer a node to be promoted into a control plane node, you can refer to the [document](https://fix.me) for how to do that.
+
+    If you don't have worker nodes in the cluster, you need to add a new node to the cluster as soon as possible. This ensures the number of control plane nodes is 3 and the control plane quorum can be formed if one of the control plane nodes fails.
+
+- There are only 2 control plane nodes in the cluster and there are no worker nodes
+
+    Removing a control plane node in this situation is not recommended because etcd data is not replicated in a single-node cluster and any failing node brings down the Harvester cluster because etcd loses quorum.
+
+
+### 2. Check volume status
+
+- Go to the `Volume` page of Longhorn GUI.
+- Check all volumes are healthy.
+
+### 3. Evict replicas from the deleting node
+
+- Go to the **Node** page of the Longhorn GUI.
```

Comment:
We can give a link here to show how to access Longhorn GUI.

https://docs.harvesterhci.io/v1.2/troubleshooting/harvester#access-embedded-rancher-and-longhorn-dashboards

Reply:
Fixed with Jillian's help.